tests/integration_tests/utils.py

```diff
@@ -137,6 +137,37 @@ def image_feature(folder, **kwargs):
     img_feature.update(kwargs)
     return img_feature
 
+def audio_feature(folder, **kwargs):
+    audio_feature = {
```

Comment:
this looks like a really big model for a test, also the 3 seconds file look pretty big, maybe all these parameters can be reduced for the sake of testing

Reply:
325a55a